storage/innobase/row/row0mysql.cc

```diff
@@ -1963,6 +1993,49 @@ row_update_for_mysql(row_prebuilt_t* prebuilt)
 	thr->fk_cascade_depth = 0;
 
 run_again:
+	if (vers_set_fields)
+	{
+		/* System Versioning: modify update vector to set
+		   sys_trx_start (or sys_trx_end in case of DELETE)
+		   to current trx_id. */
+		dict_table_t* table = node->table;
+		dict_index_t* clust_index = dict_table_get_first_index(table);
+		upd_t* uvect = node->update;
+		upd_field_t* ufield;
+		dict_col_t* col;
+		unsigned col_idx;
+		if (node->is_delete || vers_delete) {
```

Comment:
Use `node->is_deleting()` that I suggested in a different comment. It is very confusing to have aliases `vers_delete = prebuilt->upd_node->vers_delete` and `node = prebuilt->upd_node`.

Reply:
Answered [here](https://github.com/MariaDB/server/pull/478#discussion_r152949147).